php-nginx/README.md

```diff
@@ -224,18 +224,27 @@ php.ini):
 
 - BCMath (bcmath)
 - Calendar (calendar)
+- Cassandra (cassandra, third-party)
+- Ev (ev)
+- Event (event)
 - Exif (exif)
 - FTP (ftp)
 - GD (gd; with PNG, JPEG and FreeType support)
 - gettext (gettext)
-- gRPC (shared)
+- gRPC (grpc)
 - intl (intl)
+- ImageMagick (imagick)
 - mbstring (mbstring)
-- memcached (shared, enabled by default, experimental support for PHP7)
-- mongodb (shared)
+- memcached (enabled by default, experimental support for PHP7)
```

Comment:
I think we can remove `experimental support for PHP7` because we started using a stable versionof memcached from pecl